make_mock_github_containers.sh

```diff
@@ -0,0 +1,18 @@
+# you need to login first to github with your PAT
```

Comment:
For what is this sh file used?

Reply:
This is for me to test now and set-up some minimal docker containers in the ghrc